remappings.txt

```diff
@@ -1,5 +1,4 @@
 ds-test/=node_modules/ds-test/src
```

Comment:
same here, should we remove this?

Reply:
I tried, it doesn't work

Reply:
> but why?

because of the import in `Test.sol`
<img width="361" alt="image" src="https://github.com/defi-wonderland/solidity-foundry-boilerplate/assets/107591874/7be52935-7fce-49f2-a206-400edabce127">